CHANGES.txt

```diff
@@ -5,11 +5,17 @@ Features
 --------
 
 * Use ``PRETTY_URLS`` by default on all sites (Issue #1838)
+* `also_create_classifications_from_other_languages` is deprecated
```

Comment:
Nuke the setting altogether. I doubt there are any third-party taxonomy plugins that depend on it.